### PR TITLE
InfoVetted (Certified connector)

### PR DESCRIPTION
**Pull request by stephenstroud**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

### CODE REVIEW

**Comment by stephenstroud** [comment]
I opened two PRs here as was unsure which was the correct route due to merge conflicts on the first. I followed the instructions as for the below PR and then followed what others did by Raising a PR on my forked Dev branch. Please close the incorrect one.

https://github.com/microsoft/PowerPlatformConnectors/pull/3519

File: certified-connectors/InfoVetted/readme.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Title
```
Comment (vmanoharas) [inline certified-connectors/InfoVetted/readme.md]: Kindly modify the readme.md file as per the below link, please ensure that all the required sections are added. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

**Review by vmanoharas — COMMENTED** [review]
Hello @stephenstroud, 

Kindly fix the review comments.

File: certified-connectors/InfoVetted/readme.md (hunk 1 above)
Comment (stephenstroud) [reply certified-connectors/InfoVetted/readme.md]: @vmanoharas  Reviewed and updated.

Comment (vmanoharas) [reply certified-connectors/InfoVetted/readme.md]: Hello @stephenstroud, 

Still the Deployment Instructions section is missing, which is required section. Kindly update.

Comment (stephenstroud) [reply certified-connectors/InfoVetted/readme.md]: @vmanoharas I have no idea what to put here, Isn't it just installing the connector from the power automate platform? I looked at a few of the other connectors and couldn't see they've supplied this section. Please advise

**Comment by vmanoharas** [comment]
Hello @stephenstroud, 

I have copied some examples of Readme.md file from other connectors, please refer and update. 

https://github.com/reversinglabs/PowerPlatformConnectors/blob/c98975ba50e8059f560180dc6cc55311fa708624/certified-connectors/ReversingLabs%20A1000/README.md

https://github.com/microsoft/PowerPlatformConnectors/pull/3518/files#diff-6a84b0459b1b55c849b558ae7aa74fb85a86c044e63db3218ff9326a41022f6f

https://github.com/simenums/PowerPlatformConnectors/blob/c9bc28ac5e6851db052b73f5c2db11980b7f79bb/certified-connectors/Weavo%20Liquid%20Loom/readme.md

https://github.com/sfc-gh-cnayak/PowerPlatformConnectors/blob/94dd0c6d086a7a51bed37b92917fd746608de992/certified-connectors/Snowflake%20v2/readme.md

https://github.com/microsoft/PowerPlatformConnectors/pull/3509/files#diff-e2d852e4ab735e4e978bbf7f8ec468b82d69711b0a6843b1c74abe0912680ed6

**Comment by stephenstroud** [comment]
> Hello @stephenstroud,
> 
> I have copied some examples of Readme.md file from other connectors, please refer and update.
> 
> https://github.com/reversinglabs/PowerPlatformConnectors/blob/c98975ba50e8059f560180dc6cc55311fa708624/certified-connectors/ReversingLabs%20A1000/README.md
> 
> https://github.com/microsoft/PowerPlatformConnectors/pull/3518/files#diff-6a84b0459b1b55c849b558ae7aa74fb85a86c044e63db3218ff9326a41022f6f
> 
> https://github.com/simenums/PowerPlatformConnectors/blob/c9bc28ac5e6851db052b73f5c2db11980b7f79bb/certified-connectors/Weavo%20Liquid%20Loom/readme.md
> 
> https://github.com/sfc-gh-cnayak/PowerPlatformConnectors/blob/94dd0c6d086a7a51bed37b92917fd746608de992/certified-connectors/Snowflake%20v2/readme.md
> 
> https://github.com/microsoft/PowerPlatformConnectors/pull/3509/files#diff-e2d852e4ab735e4e978bbf7f8ec468b82d69711b0a6843b1c74abe0912680ed6

@vmanoharas Updated, thanks

**Review by vmanoharas — APPROVED** [review]
Dear Partner, 

Congratulations your PR is approved !! We are proud to announce the Brand-New Certification experience for you to certify Power Platform Copilot Connectors & Plugins. Read the blog here- [Announcing Partner Center to certify and publish Power Platform Copilot connectors and plugins](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fpowerautomate.microsoft.com%2Fen-us%2Fblog%2Fannouncing-partner-center-to-certify-and-publish-power-platform-copilot-connectors-and-plugins%2F&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390444301%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=5MYg0GlCsic2JqzwJU0h7mIOc7uHOeL7ZWSbgq%2FFkWk%3D&reserved=0)

As next steps: 

1.	Learn about the new certification experience- [Get your Power Platform connector and plugin certified - Overview](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fsubmit-certification&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390456966%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=uWDvs%2BmTbozw1oPc%2FNDNWJ%2FHgnUSfFlRCB1T8UHBh3A%3D&reserved=0)
2.	Package your connector and plugin files- [Prepare Power Platform connector and plugin files for certification](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-submission&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390465418%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=pDuv4bnya1eP3Rrs3p4ArNOXw%2FQipWU87q5oZ0VSIe0%3D&reserved=0)
3.	Initiate a certification request in Partner Center- [Verified publisher certification process](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fsubmit-for-certification&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390471799%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=zbm2px5VZ2ABLBFcORH6tu1fsC9KWdXkP1XJFyluMzw%3D&reserved=0)
4.	Ensure your connector & plugin files comply to the Marketplace policies here- [5000 Power Platform Connector Policies for Marketplace](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Flegal%2Fmarketplace%2Fcertification-policies%235000-power-platform-connectors&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390477736%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=h0pI5J5buo82xxvnBVmrAN8K%2F%2BK8xZlUdcSwzWoqWq0%3D&reserved=0)
5.	Test your connector post certification to provide go-live for deployment- [Testing Guidelines](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-testing&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390483743%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=Z8pSnhCArFRzEr373o29qM5h5VbV1nlN2gVbv%2Blk1aE%3D&reserved=0)
6.	If you wish to Update your connector or plugin in future- [Updating Guidelines](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-updates&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390489681%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=CvxTWJFE3bTFeJ3nDJoCSpzvimhVV9go4cBG%2FQ4gMvo%3D&reserved=0)



Many thanks,
Power Platform 
Copilot connector & plugin certification team

**Comment by stephenstroud** [comment]
@vmanoharas When following step 3. Initiate a certification request in partner Center. The offer option for "Power Platform Connector" under "Microsoft 365 and Copilot" for connector is missing. Should this offer be added under something else like the Commercial marketplace?

Partner center markerplace
image

Documentation link

https://learn.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification

image

Commercial marketplace options for reference

image


thanks

**Comment by vmanoharas** [comment]
Hello @stephenstroud, 

Please provide the Seller ID of your partner center account which is associated with Microsoft 365 and Copilot program.

**Comment by stephenstroud** [comment]
> Hello @stephenstroud,
> 
> Please provide the Seller ID of your partner center account which is associated with Microsoft 365 and Copilot program.

@vmanoharas  SellerId: 87510750

I have also submitted for certification through the isvstudio portal.

**Comment by vmanoharas** [comment]
> > Hello @stephenstroud,
> > Please provide the Seller ID of your partner center account which is associated with Microsoft 365 and Copilot program.
> 
> @vmanoharas SellerId: 87510750
> 
> I have also submitted for certification through the isvstudio portal.

Thank you @stephenstroud, we will get back to you in 2 to 3 day.

**Comment by stephenstroud** [comment]
@vmanoharas  Any update? thanks

**Comment by vmanoharas** [comment]
> @vmanoharas Any update? thanks

Hello @stephenstroud, 

Please accept my apology for the delay.  Could you please send your seller ID to the below email for further discussion. 

For any queries or concerns please get in touch with us on certificationteam@microsoft.com

**Comment by stephenstroud** [comment]
@vmanoharas  Certification submitted, any idea how long the approval takes? thanks

**Comment by vmanoharas** [comment]
> @vmanoharas Certification submitted, any idea how long the approval takes? thanks

@stephenstroud, approximately, it would take 6 to 7 weeks.

**Comment by stephenstroud** [comment]
> > @vmanoharas Certification submitted, any idea how long the approval takes? thanks
> 
> @stephenstroud, approximately, it would take 6 to 7 weeks.

Okay thanks, may need a new access token then....

**Pull request merged**